RELEASING/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+SUPERSET_RC_VERSION ?= $(shell bash -c 'read -p "Superset RC Version: " rc_version; echo $$rc_version')
+APACHE_USER_EMAIL ?= $(shell bash -c 'read -p "Apache user email: " user_email; echo $$user_email')
+GET_TOKEN ?= $(shell bash -c 'read -p "Github token: " github_token; echo $$github_token')
+$(eval dry-run:;@:)
+$(eval e-break:;@:)
+
+install:
+	pip install -r RELEASING/requirements.txt
+
+setup:
+	. set_release_env.sh $(SUPERSET_RC_VERSION) $(APACHE_USER_EMAIL)
+
+cherry-tree:
+ifndef GITHUB_TOKEN
+	$(eval TOKEN=$(GET_TOKEN))
+else
+	$(eval TOKEN=${GITHUB_TOKEN})
+endif
+
+ifeq (dry-run, $(filter dry-run,$(MAKECMDGOALS)))
+	$(eval DRY_RUN=-nd)
+endif
+
+ifeq (e-break, $(filter e-break,$(MAKECMDGOALS)))
+	$(eval ERROR_MODE_BREAK=--error-mode break)
+endif
+	cherrytree bake -r apache/superset -m master -l v${SUPERSET_GITHUB_BRANCH} ${DRY_RUN} ${ERROR_MODE_BREAK} ${SUPERSET_GITHUB_BRANCH} --access-token ${TOKEN}
+
+cut-branch:
```

Comment:
# TODO: I'll follow up and add a warning here if the github environments are not defined.